compiler/resolution/fcf-support.cpp

```diff
@@ -149,8 +152,9 @@ attachChildThis(const SharedFcfSuperInfo info, AggregateType* child,
                 FnSymbol* payload);
 
 static FnSymbol*
-attachChildWriteThis(const SharedFcfSuperInfo info, AggregateType* child,
-                     FnSymbol* payload, const char* name);
+attachChildWriteMethod(const SharedFcfSuperInfo info, AggregateType* child,
+                       FnSymbol* payload,
+                       const char* name);
```

Comment:
Seems a little odd that these are on different lines when the first line has two arguments on it, but it's not a big deal by any means

Reply:
I always thought the old compiler did the strategy of "as many args on the first line as possible, then one-per subsequent line" which is why I've done things this way - but I've also been around long enough to not be surprised that I could be wrong or that the old compiler could have no standard xD.

Reply:
I think the old compiler just didn't have a standard XD At times it's been "what fits", sometimes it's been "line them up so that the arg names all start at the same spot", sometimes "one arg per line"